inc/Engine/Admin/Beacon/Beacon.php

```diff
@@ -81,12 +81,12 @@ public function insert_script() {
 		}
 
 		echo '<script type="text/javascript">!function(e,t,n){function a(){var e=t.getElementsByTagName("script")[0],n=t.createElement("script");n.type="text/javascript",n.async=!0,n.src="https://beacon-v2.helpscout.net",e.parentNode.insertBefore(n,e)}if(e.Beacon=n=function(t,n,a){e.Beacon.readyQueue.push({method:t,options:n,data:a})},n.readyQueue=[],"complete"===t.readyState)return a();e.attachEvent?e.attachEvent("onload",a):e.addEventListener("load",a,!1)}(window,document,window.Beacon||function(){});</script>
-			<script type="text/javascript">window.Beacon(\'init\', \'' . esc_js( $form_id ) . '\')</script>
-			<script>window.Beacon("identify", ' . wp_json_encode( $this->identify_data() ) . ');</script>
-			<script>window.Beacon("session-data", ' . wp_json_encode( $this->session_data() ) . ');</script>
-			<script>window.addEventListener("hashchange", function () {
-				window.Beacon("suggest");
-			  }, false);</script>';
+		<script type="text/javascript">window.Beacon(\'init\', \'' . esc_js( $form_id ) . '\')</script>
+		<script>window.Beacon("identify", ' . wp_json_encode( $this->identify_data() ) . ');</script>
+		<script>window.Beacon("session-data", ' . wp_json_encode( $this->session_data() ) . ');</script>
+		<script>window.addEventListener("hashchange", function () {
+			window.Beacon("suggest");
+		}, false);</script>';
```

Comment:
This code is mixing concerns. I'm wondering if we want to move it to a view file. Thoughts?

Reply:
I agree, adding a view for it makes a lot of sense. I went ahead and did that in https://github.com/wp-media/wp-rocket/pull/2616/commits/4d440f8a929f436d4a5f14009726fc98819b6ab5